utils/utils.go

```diff
@@ -0,0 +1,25 @@
+package utils
+
+import (
+	"os"
+	"strconv"
+
+	pbbstream "github.com/streamingfast/bstream/pb/sf/bstream/v1"
+)
+
+func GetEnvForceFinalityAfterBlocks() *uint64 {
+	if fin := os.Getenv("FORCE_FINALITY_AFTER_BLOCKS"); fin != "" {
+		if fin64, err := strconv.ParseInt(fin, 10, 64); err == nil {
+			finu64 := uint64(fin64)
+			return &finu64
+		}
+	}
+	return nil
+}
+
+func TweakBlockFinality(blk *pbbstream.Block, maxDistanceToBlock uint64) {
+	if (blk.Number >= maxDistanceToBlock) &&
```

Comment:
Shouldn't you actually have something like 
```suggestion
    blockCountSinceGenesis := blk.Number - bstream.FirstStreamableBlock`
	if (blockCountSinceGenesis >= maxDistanceToBlock) &&
```

Reply:
No, it's not that function's role to adjust for the FirstStreamableBlock.
We just don't want to underflow the uint64. 

Clarifying the intent here:
```
func TweakBlockFinality(blk *pbbstream.Block, maxDistanceToBlock uint64) {
	if blk.LibNum > blk.Number {
		panic("libnum cannot be greater than block number")
	}
	if blk.LibNum < maxDistanceToBlock {
		return // don't touch blocks at the beginning of the chain
	}
	if (blk.Number - blk.LibNum) >= maxDistanceToBlock {
		blk.LibNum = blk.Number - maxDistanceToBlock // force finality
	}
}```

Reply:
Why don't clarify the code also, or maybe you are doing that right now :)